src/DataSource/Http.hs

```diff
@@ -16,6 +16,7 @@ import           Control.Monad.Reader (ask)
 import           Data.Aeson (parseJSON)
 import           Data.Aeson.Text (encodeToLazyText)
 import           Data.List (nub)
+import           Data.Text (pack)
```

Comment:
You don't need `pack`, you can look at https://github.com/serokell/universum/blob/master/src/Universum/String/Conversion.hs